snuba/admin/static/api_client.tsx

```diff
@@ -200,6 +201,24 @@ function Client() {
       });
     },
 
+    executeSnQLQuery: (query: SnQLRequest) => {
+      const url = baseUrl + "production_snql_query";
+      return fetch(url, {
+        headers: { "Content-Type": "application/json" },
+        method: "POST",
+        body: JSON.stringify(query),
+      }).then((res) => {
+        if (res.ok) {
+          return Promise.resolve(res.json());
+        } else {
+          return res.json().then((err) => {
+            let errMsg = err?.error.message || "Could not convert SnQL";
```

Comment:
"Could not convert SnQL" is probably not the right error message.

Reply:
lol good catch